pkgs/top-level/python-packages.nix

```diff
@@ -22182,6 +22182,29 @@ in modules // {
     };
   };
 
+  reno = buildPythonPackage rec {
+    name = "reno-${version}";
+    version = "1.8.0";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/r/reno/${name}.tar.gz";
+      sha256 = "1pqg0xzcilmyrrnpa87m11xwlvfc94a98s28z9cgddkhw27lg3ps";
+    };
+
+    # Tests require gnupg1 and git
+    doCheck = false;
+
+    dontStrip = true;
+    propagatedBuildInputs = with self; [ pbr six pyyaml ];
+    buildInputs = with self; [ Babel testtools testscenarios oslotest ];
+
+    meta = {
+      description = "Release Notes Manager";
+      homepage = http://docs.openstack.org/developer/reno/;
+      license = licenses.asl20;
```

Comment:
please add yourself as maintainer. Don't forget to add yourself to lib.maintainers as well if you haven't done so yet.